nvm.sh

```diff
@@ -3227,11 +3239,13 @@ nvm_auto() {
       nvm install >/dev/null
     fi
   elif [ "_$NVM_MODE" = '_use' ]; then
-   VERSION="$(nvm_resolve_local_alias default 2>/dev/null || nvm_echo)"
-    if [ -n "$VERSION" ]; then
-      nvm use --silent "$VERSION" >/dev/null
-    elif nvm_rc_version >/dev/null 2>&1; then
-      nvm use --silent >/dev/null
+    if [ "_$NVM_CURRENT" = '_system' ]; then
```

Comment:
Thanks! Perhaps `nvm_auto` is the better place - but then it'd probably be best done at the top of the function, not just in the "use" path.